master/getting-started/docker/tutorials/security-using-docker-labels-and-calico-policy.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Security using Docker Labels and Calico Policy
+---
+
+## Background
+
+With Calico as a Docker network plugin, Calico can be configured to extract the
+labels on a container and apply them to the workload endpoint for use with Calico
+[policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy).
+All traffic must be explicitly allowed through configuration of the globally
```

Comment:
I don't want people to think "All traffic must be explicitly allowed" is a prerequisite or anything. Can you add in "By default, Calico blocks all traffic unless it has been explicitly allowed..."